src/pages/Designer/Designer.js

```diff
@@ -127,13 +127,14 @@ const Designer = ({ update, viewMode = false, sharedMetadata = undefined }) => {
     const [projectNames, setProjectNames] = useState([]);
     const [defaultProjectName, setDefaultProjectName] = useState('');
     const [credits, setCredits] = useState(0);
-    const [userCredits,setUserCredits]=useState(0);
+    const [userCredits, setUserCredits] = useState(0);
     const [aiServices, setAiServices] = useState([]);
     const reactFlowWrapper = useRef(null);
     const edgeUpdateSuccessful = useRef(true);
     const toastIdRef = useRef();
     const ref = useRef(null);
     const [spinner, setSpinner] = useState(false);
+    const [buildTool, setBuildTool] = useState('maven');
```

Comment:
Not Required as the build tool is being set at the application data level.

Reply:
Removed the parameter